### PR TITLE
Sf 1162 Welcome message as a modified inline alert until user deposits funds

### PR DESCRIPTION
**Pull request by leeshuzheng**

### CODE REVIEW

File: src/components/templates/SimpleAdvancedView/SimpleAdvancedView.tsx [hunk 1]
```diff
@@ -21,9 +24,26 @@ export const SimpleAdvancedView = ({
 
     const component = view === 'Simple' ? simpleComponent : advanceComponent;
 
+    const { isConnected } = useAccount();
+    const balance = useBalances();
+
+    const isShowWelcomeAlert =
+        Object.values(balance).every(v => v === 0) || !isConnected;
```
Comment (leeshuzheng) [inline src/components/templates/SimpleAdvancedView/SimpleAdvancedView.tsx:31]: The alert only shows if

1) wallet isn't connected
2) wallet balance is 0 for every currency

File: src/components/molecules/Alert/types.ts [hunk 2]
```diff
@@ -3,4 +3,5 @@ export enum AlertSeverity {
     Info = 'info',
     Success = 'success',
     Warning = 'warning',
+    Basic = 'basic',
```
Comment (leeshuzheng) [inline src/components/molecules/Alert/types.ts:6]: After discussion with @benk-sf , this new variant will be used in portfolio management too.

No alert icon and no close button (the old behaviour) @as472

File: src/components/templates/SimpleAdvancedView/SimpleAdvancedView.tsx (hunk 1 above)
Comment (as472) [reply src/components/templates/SimpleAdvancedView/SimpleAdvancedView.tsx:31]: 2. walletBalance is 0 or collateralBalance is 0?

File: src/components/molecules/Alert/Alert.stories.tsx [hunk 3]
```diff
@@ -42,6 +42,13 @@ Success.args = {
     severity: AlertSeverity.Success,
 };
 
+export const Basic = Template.bind({});
```
Comment (as472) [inline src/components/molecules/Alert/Alert.stories.tsx:45]: can you fix the stories in this PR for only 2 break points?

File: src/components/molecules/Alert/Alert.tsx [hunk 4]
```diff
@@ -88,19 +90,21 @@ export const Alert = ({
...
-                    'h-[13px] w-[13px] flex-shrink-0 laptop:h-4 laptop:w-4',
-                    {
-                        'mt-[3px] laptop:mt-0.5': !subtitle,
-                    }
-                )}
-                data-testid='close-button'
-            >
-                <XMarkIcon />
-            </button>
+            {isShowCloseButton && (
```
Comment (as472) [inline src/components/molecules/Alert/Alert.tsx:93]: please add test for this change

File: src/components/molecules/Alert/Alert.stories.tsx (hunk 3 above)
Comment (leeshuzheng) [reply src/components/molecules/Alert/Alert.stories.tsx:45]: Will do

File: src/components/templates/SimpleAdvancedView/SimpleAdvancedView.tsx (hunk 1 above)
Comment (leeshuzheng) [reply src/components/templates/SimpleAdvancedView/SimpleAdvancedView.tsx]: Wallet Balance

**Pull request merged**